include/seqan3/alignment/all.hpp

```diff
@@ -10,20 +10,295 @@
+ *<td style="background: #90ff90; color: black; vertical-align: middle; text-align: center; border: 1px solid black;" class="table-yes"> ✓ </td>
+ *<td style="background: #90ff90; color: black; vertical-align: middle; text-align: center; border: 1px solid black;" class="table-yes"> ✓ </td>
+ *<td style="background: #90ff90; color: black; vertical-align: middle; text-align: center; border: 1px solid black;" class="table-yes"> ✓ </td>
+ *<td style="background: #90ff90; color: black; vertical-align: middle; text-align: center; border: 1px solid black;" class="table-yes"> ✓ </td>
+ *<td style="background: #90ff90; color: black; vertical-align: middle; text-align: center; border: 1px solid black;" class="table-yes"> ✓ </td>
+ *<td style="background: #90ff90; color: black; vertical-align: middle; text-align: center; border: 1px solid black;" class="table-yes"> ✓ </td>
+ *<td style="background: #90ff90; color: black; vertical-align: middle; text-align: center; border: 1px solid black;" class="table-yes"> ✓ </td>
+ *<td style="background: #ff9090; color: black; vertical-align: middle; text-align: center; border: 1px solid black;" class="table-no"> ✘ </td>
+ *</tr>
+ *</table>
+ *
+ * ## Accessing the computed alignment
+ *
+ * The seqan3::align_pairwise interface returns a seqan3::alignment_range. This range is a single pass range over
+ * the computed alignments and dereferencing the iterator over this range will return seqan3::alignment_result objects.
+ * Even if only a single alignment is computed a range will be returned since it could be possible that
+ * one alignment invocation produces multiple results, e.g. to receive suboptimal alignments.
+ * The seqan3::alignment_result object contains only the information that have been requested by the alignment
+ * configuration. The seqan3::align_cfg::result configuration element allows to limit the parts that are computed
+ * in the alignment algorithm. The following table shows the parts that are computed depending on the
+ * seqan3::align_cfg::result configuration:
+ *
+ * | **Entity**                                                                     | **Available result**                                  |
+ * | -------------------------------------------------------------------------------|------------------------------------------------------ |
+ * | \ref seqan3::align_cfg::result::with_score  "with_score"                       | alignment score                                       |
+ * | \ref seqan3::align_cfg::result::with_back_coordinate  "with_back_coordinate"   | alignment score; back coordinate                      |
+ * | \ref seqan3::align_cfg::result::with_front_coordinate  "with_front_coordinate" | alignment score; back and front coordinate            |
+ * | \ref seqan3::align_cfg::result::with_alignment  "with_alignment"               | alignment score; back and front coordinate; alignment |
```

Comment:
Again, are you sure this is resolved correctly in doxygen?

Reply:
yes. That's why I didn't use the MD table, because otherwise it looks ugly and ref didn't work. But I try it again

Reply:
Yes it does